Remove outdated comments

jarsi opened the pull request:
Going through the code I found some comments and commented out code snippets which seem to be either outdated or superflous. This PR addresses these snippets.﻿

albada commented:
It looks good to me except two things:

- fac could be removed altogether, as its value is 1.

- I don't understand the condition 
if mu <= V_th + (0.95 * abs(V_th) - abs(V_th)):
This should be simplified, and I would also compare it with what is used in the mean-field toolbox.

jarsi commented:
I removed fac and simplified the condition. The condition as it was before matched [lif_meanfield_tools](https://github.com/INM-6/lif_meanfield_tools/blob/9627c984d45eab280f360b28d801b7f0d509201b/lif_meanfield_tools/aux_calcs.py#L85).

AlexVanMeegen commented:
The condition as it is now matches current [NEST master](https://github.com/nest/nest-simulator/blob/master/models/siegert_neuron.cpp#L339).
